editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/l10n/messages.properties

```diff
@@ -66,4 +66,9 @@ newProjectWizardTemplatePageTemplateProjectLabel = Use a predefined template
 newProjectWizardTemplatePageTemplateProjectDescription = Select this option if you would like to start with one of our predefined project templates.
 newProjectWizardTemplatePageDSLLabel = Select the project type
 newProjectWizardTemplatePageFilterBoxText = type filter text (filters on name, description and keywords)
+StagingRepositoriesFieldEditor_ErrorMessageDuplicateRepoId=Two repositories have the same name: 
+StagingRepositoriesFieldEditor_ErrorMessageWrongPattern=The staging repositories need to follow the pattern <repositoryName1>,<repositoryURL1>;<repositoryName2>,<repositoryURL2>

```

Comment:
I would change this warning text slightly to- "Staging repositories must follow the pattern <repositoryName1>,<repositoryURL1>;<repositoryName2>,<repositoryURL2>" just to reduce wordiness a bit.